sympy/stats/joint_rv_types.py

```diff
@@ -253,3 +253,71 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
     A random symbol
     """
     return multivariate_rv(NormalGammaDistribution, syms, mu, lamda, alpha, beta)
+
+
+#-------------------------------------------------------------------------------
+# Multivariate Beta distribution ---------------------------------------------------------
+
+class MultivariateBetaDistribution(JointDistribution):
+
+    _argnames = ['alpha']
+    is_Continuous = True
+
+    def check(self, alpha):
+        _value_check(len(alpha) >= 2, "At least two categories should be passed.")
+        _value_check(all([a_k > 0 for a_k in alpha]), "Each concentration parameter"
+                                                        " should be positive")
+
+    @property
+    def set(self):
+        k = len(self.alpha)
+        return Interval(0, 1)**k
+
+    def pdf(self, *syms):
+        alpha = self.alpha
+        B = Mul(*[gamma(a_k) for a_k in alpha])/gamma(Add(*[a_k for a_k in alpha]))
```

Comment:
This could possibly be simplified.
```suggestion
        B = Mul(*list(map(gamma, alpha)))/gamma(Add(*alpha))
```